### PR TITLE
CON-71 Send notification to admins when an event is auto cancelled

### PR DESCRIPTION
**Pull request by codinger41**
#### Description
This Pull request allows an administrator for the converge web app to receive notifications whenever an event has been auto cancelled.

#### How has this been tested
- [x] Unit tests
- [x] New feature (non-breaking change which adds functionality
- [x] Integration tests

#### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/andela/mrm_api.git`
2. Setup project according to Readme.md
3. checkout to branch `story/CON-71-admin-autocancellation-notification`
4. run the following query to cancel an event. Make sure the eventId and other details are valid:

```
mutation {
 	cancelEvent(
    eventId: "041t6mhogr8cq7i8vvr2h36d9p",
    calendarId: "andela.com_38393338303730373538@resource.calendar.google.com",
    eventTitle: "First LFA On-Boarding",
    startTime: "2018-08-14T06:00:00-07:00",
    endTime: "2018-08-14T07:00:00-07:00"
  ) {
    event {
      id
      eventTitle
    }
  }
}
```

5. Run the query below to get all admin notifications:

```

query {
	allUnreadNotifications{
    notifications {
      title
      status
      id
    }
  }
}

```

6. Run the query below to change the status of a notification from `unread` to `read`:

```
mutation {
  updateNotificationStatus(notificationId: 26){
    notification {
      id
      title
      message
    }
  }
}
```

7. To test the sockets, create an HTML file `index.html` and fill it with the following HTML code and change the location in the channel string to whatever location the cancelled event is happening, ie from `notificatons-lagos` to `notifications-kampala`

```
<!DOCTYPE html>
  <html lang="en">
  <head>
    <title>Flask_Chat_App</title>
    <script
      src="https://code.jquery.com/jquery-3.4.1.min.js"
      integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
      crossorigin="anonymous"></script>
  </head>
  <body>

    <h3 id="#messages" style='color: #ccc;font-size: 30px;'>No message yet..</h3>
    <div class="message_holder"></div>

    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/1.7.3/socket.io.min.js"></script>
    <script type="text/javascript">
      $(document).ready(function() {
        var socket = io.connect('http://127.0.0.1:5000');
        socket.on('connect', function() {
          socket.send('User has connected!');
        });
        socket.on('notifications-lagos', function(msg) {
          console.log(msg)
          console.log('Received message');
        });
      });
    </script>
  </body>
  </html>
```

8. After creating this HTML file, open it in the browser and open the console,  fire the query to cancel another event and you'd see the event object in the console of the HTML page.

#### Checklist
- [x] My changes generate no new warnings
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] Existing unit tests pass locally with my changes
- [x] Implementation works according to expectations

#### JIRA
[CON-72](https://andela-apprenticeship.atlassian.net/jira/software/projects/CON/boards/15/backlog?label=backend&selectedIssue=CON-71)

### CODE REVIEW

**Comment by codinger41** [comment]
Thanks @mnswaleh I'll do that

File: admin_notifications/helpers/create_notification.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import celery
+
+
+def update_notification(notification_id):
+    notification = AdminNotification.query.filter_by(id=notification_id).first()
+    notification.date_received = datetime.now()
+    notification.save()
+
+
+@celery.task
```
Comment (Arusey) [inline admin_notifications/helpers/create_notification.py]: Please specify the name of the task that is run asynchronously. Such as `@celery.task(name='asynchronous-email-notifications')
`

File: requirements.txt [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
+Flask-JSON==0.3.2
 Flask-Mail==0.9.1
+Flask-Script==2.0.6
+flask-socketio==4.1.0
+Flask-Testing==0.7.1
 google-api-python-client==1.6.7
-graphene-sqlalchemy==2.0.0
 graphene==2.1
+graphene-sqlalchemy==2.0.0
+graphql-core==2.2
```
Comment (Arusey) [inline requirements.txt]: There seem to be additional dependencies added on here...Kindly ensure to install only the dependencies you need for this task.

**Comment by rajeman** [comment]
Tasks related to admin notifications have been suspended.

**Pull request closed without merging**